dev/scripts/remove-ts-machines.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
```

Comment:
I like to start my bash script with:

```suggestion
set -euo pipefail
IFS=$'\n\t'
```

It really helps avoiding weird, hard to debug, issues with bash script. See this blog post for details: http://redsymbol.net/articles/unofficial-bash-strict-mode/